sql/fxa_users_last_seen_v1.init.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
+  submission_date,
+  CURRENT_DATETIME() AS generated_time,
+  submission_date AS date_last_seen,
+  IF(country IN ('United States', 'France', 'Germany', 'United Kingdom', 'Canada'),
+    submission_date,
+    NULL) AS date_last_seen_in_tier1_country,
+  * EXCEPT (submission_date, generated_time)
+FROM
+  fxa_users_daily_v1
+WHERE
+  -- 2017-10-01 is the first date in the Amplitude FxA project.
+  submission_date = DATE '2017-10-01'
```

Comment:
I think we should do this in the same format as `clients_last_seen_v1.init.sql`, so that it can be used to generate any date without prior days, which is useful during backfill.

```sql
WITH
  input AS (
  SELECT
    ROW_NUMBER() OVER w1_unframed AS _n,
    FIRST_VALUE(IF(country IN ('United States',
          'France',
          'Germany',
          'United Kingdom',
          'Canada'),
        submission_date,
        NULL) IGNORE NULLS) OVER w1 AS date_last_seen_in_tier1_country,
    *
  FROM
    analysis.fxa_users_daily_v1
  WHERE
    submission_date <= @submission_date
    AND submission_date > DATE_SUB(@submission_date, INTERVAL 28 DAY)
  WINDOW
    w1 AS (
    PARTITION BY
      user_id
    ORDER BY
      submission_date DESC ROWS BETWEEN UNBOUNDED PRECEDING
      AND UNBOUNDED FOLLOWING),
    -- We must provide a modified window for ROW_NUMBER which cannot accept a frame clause.
    w1_unframed AS (
    PARTITION BY
      user_id
    ORDER BY
      submission_date DESC) )
SELECT
  @submission_date AS submission_date,
  CURRENT_DATETIME() AS generated_time,
  submission_date AS last_seen_date,
  * EXCEPT (_n,
    submission_date,
    generated_time)
FROM
  input
WHERE
  _n = 1
```

Reply:
We discussed face-to-face and I'm going to merge this init.sql as-is with the understanding that @relud is working on a new approach to schema definition and metadata for this repo which will make this init.sql irrelevant.